docs_source_files/content/_index.ja.md

```diff
@@ -1,35 +1,23 @@
 ---
-title: "The Selenium Browser Automation Project"
+title: "Seleniumブラウザー自動化プロジェクト"
 ---
 {{% notice info %}}
```

Comment:
@miekof could you please remove the notice? The page is already fully translated.